File: Unit14/app/src/main/java/teach/meskills/unit14/MainActivity.java
package teach.meskills.unit14;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    Button formatLetters, magicBall, riddles, calculator;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        formatLetters = (Button) findViewById(R.id.format_button);
        magicBall = (Button) findViewById(R.id.magic_ball);
        riddles = (Button) findViewById(R.id.riddles);
        calculator = (Button) findViewById(R.id.calculator);
        formatLetters.setOnClickListener(v -> {
            startActivity(new Intent(MainActivity.this, FormatLetters.class));
        });
        magicBall.setOnClickListener(v -> {
            startActivity(new Intent(MainActivity.this, MagicBall.class));
        });
        riddles.setOnClickListener(v -> {
            startActivity(new Intent(MainActivity.this, Riddles.class));
        });
        calculator.setOnClickListener(v -> {
            startActivity(new Intent(MainActivity.this, Calculator.class));
        });
    }
}

File: Unit14/app/src/main/java/teach/meskills/unit14/MainActivity.kt
package teach.meskills.unit14

import androidx.appcompat.app.AppCompatActivity
import android.widget.Button
import android.os.Bundle
import android.content.Intent

class MainActivity : AppCompatActivity() {
    private lateinit var formatLetters: Button
    private lateinit var magicBall: Button
    private lateinit var riddles: Button
    private lateinit var calculator: Button
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main_activity)
        formatLetters = findViewById(R.id.format_button)
        magicBall = findViewById(R.id.magic_ball)
        riddles = findViewById(R.id.riddles)
        calculator = findViewById(R.id.calculator)
        formatLetters.setOnClickListener {
            startActivity(
                Intent(
                    this, FormatLetters::class.java
                )
            )
            // Новый комит для тренировки
        }
        magicBall.setOnClickListener {
            startActivity(
                Intent(
                    this, MagicBall::class.java
                )
            )
        }
        riddles.setOnClickListener {
            startActivity(
                Intent(
                    this, Riddles::class.java
                )
            )
        }
        calculator.setOnClickListener {
            startActivity(
                Intent(
                    this, Calculator::class.java
                )
            )
        }
    }
}